### PR TITLE
Removes roundstart announcements

### PR DESCRIPTION
**Pull request by private-tristan**
# About the pull request

Marines will no longer receive an announcement about what's going on when they wake up, leaving it solely to the PLTCO to inform them of what's happening.

also removes the ARES ONLINE text

# Explain why it's good for the game

morrow had this listed on the to-do sheet
image

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Private Tristan
del: Marines no longer get an announcement at roundstart about the colony going dark.
del: ARES no announces itself at roundstart.
/:cl:

### CODE REVIEW

**Comment by private-tristan** [comment]
linters failed but PR works fine.
image

**Comment by harryob** [comment]
> linters failed but PR works fine. image

click in annotate lints

CleanShot 2023-11-26 at 20 35 18@2x

File: code/datums/emergency_calls/emergency_call.dm [hunk 1]
```diff
@@ -10,11 +10,6 @@
 	var/datum/emergency_call/picked_calls[] = list() //Which distress calls are currently active
 	var/ert_dispatched = FALSE
 
-/datum/game_mode/proc/ares_online()
```
Comment (morrowwolf) [inline code/datums/emergency_calls/emergency_call.dm:13]: Just override these functions on the ai gamemode please, leave everything else so we can avoid conflicts

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by private-tristan** [comment]
going to make a different branch for this because github desktop is being difficult.

**Pull request closed without merging**